README.md

```diff
@@ -107,6 +107,14 @@ Use Skatepark to develop, test, and validate that your program will operate as e
 > [!IMPORTANT]
 > The client program is an _example_ for how to interact with the Managed Structure's API. It is useful for testing your Managed Structure locally, but ultimately you will want to integrate your Managed Structure with your own application. 
 
+
+# Simulating Structure Run Delay
+By default, Skatepark immediately transitions Structure Runs from the `QUEUED` state to the `RUNNING` state.
```

Comment:
does this mean that the client has a high probability of missing the QUEUED state? If so, I would rather default the value to something that reduces or eliminates that probability (is 1 second enough?) to give additional confidence that QUEUED is caught. Allow them to override it to zero if they want the instant shift change.

Reply:
Good point `1` is probably fine